paymentsheet/src/main/java/com/stripe/android/paymentsheet/analytics/PaymentSheetEvent.kt

```diff
@@ -116,10 +148,9 @@ internal sealed class PaymentSheetEvent : AnalyticsEvent {
     }
 
     class Dismiss(
-        mode: EventReporter.Mode,
         isDecoupled: Boolean,
     ) : PaymentSheetEvent() {
-        override val eventName: String = formatEventName(mode, "dismiss")
+        override val eventName: String = "mc_dismiss"
```

Comment:
iOS is sending `mc_dismiss`, but Android has historically sent `mc_{mode}_dismiss`. Should we change iOS to also include the mode (given that iOS hasn’t historically sent this event)?

Reply:
Hmm, this is unfortunate. I don't particularly find much value in the mode w.r.t to the dashboards we are building today, and we already have 2 iOS SDK versions sending just "mc_dismiss", so I'm not super thrilled with the idea of changing it on either platform.

My general take is that encoding data into the event name is an anti-pattern and we should log mode as an actual parameter in the request.

I'm peaceful with leave as-is on Android and having this discrepancy between the platforms, since I think we will have separate queries per platform anyways.

Reply:
Nevermind! After writing this comment, I realized we have never sent this event, so I’m good with just using `mc_dismiss` and omitting the mode.